api/src/utils/jwt.js

```diff
@@ -27,7 +27,9 @@ class JWT {
   }
 }
 
-exports.JWT = JWT;
+const DEFAULT_JWT_TOKEN_EXPIRY = 3600;
```

Comment:
I see in pr description you mentioned 7 days, but here default expiration time is one hour.